File: r/hello.r
my.name <- readline(prompt="Enter name: ")
my.age <- readline(prompt="Enter age: ")
my.age <- as.integer(my.age)
print(paste("Hi,", my.name, " Weclome to R programming. Next year you will be", my.age+1, "years old."))

add <- function(x, y) {
return(x + y)
}
subtract <- function(x, y) {
return(x - y)
}
multiply <- function(x, y) {
return(x * y)
}
divide <- function(x, y) {
return(x / y)
}

File: java/hello.java
import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
	    

    // create a scanner so we can read the command-line input
    Scanner scanner = new Scanner(System.in);

    System.out.print("Enter your name: ");
    String username = scanner.next();

    System.out.print("Enter your age: ");
    int age = scanner.nextInt();

    System.out.println(String.format(" Hello %s, Welcome to Java programming. By the way you are %d old", username, age));
	
	    
	}
}
  

